code/game/gamemodes/wizard/wizloadouts.dm

```diff
@@ -71,11 +71,11 @@
 	desc = "A set of legendary artifacts used by a bald, grey wizard, now passed on to you. <br> \
 		Open His Grace's latch once you are ready to kill by using It in your hand. Keep It fed or you will be Its next meal.<br> \
 		You might want to raid the Armory or loot a Security Officer to get ranged weapons like a disabler, His Grace's Hunger has little patience.<br><br> \
```

Comment:
They now have a ranged offensive spell, good idea to tweak this description?

Reply:
This better?

Reply:
When is the holy toolbox not enough? 😄